ViaEventAssociation.Core.Domain/IEventRepository.cs

```diff
@@ -0,0 +1,11 @@
+﻿using ViaEventAssociation.Core.Domain.Aggregates.Events;
+using ViaEventAssociation.Core.Domain.Aggregates.Events.ValueObjects;
+using ViaEventAssociation.Core.Tools.OperationResult;
+
+namespace ViaEventAssociation.Core.Domain;
+
+public interface IEventRepository
+{
```

Comment:
maybe we can have  public Task<Result<ViaEvent>>(EventId eventId) if we want asyncrounous call to db